source/_integrations/lamarzocco.markdown

```diff
@@ -90,19 +90,19 @@ Use Bluetooth:
 
 ## Switches
 
-| Switch name | Description | Available for machines |
-|-------------|-------------| ---------------------- |
-| Main        | Allows to turn machines on-/off | all |
-| Steam boiler | Allows to enable/disable the steam boiler | all |
-| Smart standby enabled | Whether smart standby is on (machine will automatically stand by after given time) | all |
+| Switch name | Description | Available for machines | Controllable through |
+|-------------|-------------| ---------------------- | ------|
+| Main        | Allows to turn machines on-/off | all | <iconify-icon inline title="Cloud" icon="material-symbols:cloud"></iconify-icon> <iconify-icon inline title="Bluetooth" icon="material-symbols:bluetooth"></iconify-icon> |
```

Comment:
Icons should be used like this:

```suggestion
| Main        | Allows to turn machines on-/off | all | {% icon "material-symbols:cloud" %} {% icon "material-symbols:bluetooth" %} |
```

Reply:
thanks, forgot to set to draft 🤦‍♂️

Reply:
how do you give it a title (used for accessibility/hover) in that syntax?

Reply:
there's another problem with that  syntax: for `bluetooth` the linter fails that Bluetooth should be written upper case

Reply:
> how do you give it a title (used for accessibility/hover) in that syntax?

It doesn't support that, but I'm open for help with extending that specific plugin.

> there's another problem with that syntax: for bluetooth the linter fails that Bluetooth should be written upper case


Hmmm that seems hard to solve right now, let's use the extensive HTML syntax for those as an exception.

Reply:
done

Reply:
also extended the plugin here https://github.com/home-assistant/home-assistant.io/pull/36004